lib/ansible/modules/cloud/docker/docker_service.py

```diff
@@ -143,6 +143,13 @@
 
 requirements:
     - "python >= 2.6"
+    - "docker-py >= 1.8.0"
```

Comment:
Does `docker_service` needs an explicitly dependency on `docker-py` version, or just implicitly inherits  `docker-compose`'s requirement?

Reply:
It should implicitly inherit `docker-compose`'s requirements. My idea was to list it nonetheless (so it's listed for all `docker_*` modules), but I can remove it if you prefer.

Reply:
Yes, leaving a deprecation comment about `docker-py` and `docker` everywhere is boring, but useful for newbies.